src/main/java/online/book/store/dto/cart/item/AddBookToTheShoppingCartDto.java

```diff
@@ -0,0 +1,9 @@
+package online.book.store.dto.cart.item;
+
+import lombok.Data;
+
+@Data
+public class AddBookToTheShoppingCartDto {
+    private Long bookId;
+    private int quantity;
```

Comment:
why validation annotations are not present here? e.g.
https://github.com/NotNull, @min